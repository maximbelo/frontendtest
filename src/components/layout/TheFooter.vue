<template>
  <footer>
    <div class="container">
      <p>
        &copy;
        {{ copyright }}
        maxim.
      </p>
    </div>
  </footer>
</template>

<script setup lang="ts">
import { ref } from "vue";

const copyright = ref<number>(new Date().getFullYear());
</script>

<style lang="scss" scoped>
footer {
  margin-top: 32px;

  p {
    text-align: center;
  }
}
</style>
