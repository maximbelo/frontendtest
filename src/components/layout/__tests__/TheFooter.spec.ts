import { describe, expect, test } from "vitest";
import { mount } from "@vue/test-utils";
import TheFooter from "@/components/layout/TheFooter.vue";

describe("TheFooter.vue tests", () => {
  test("matches the snapshot", () => {
    const wrapper = mount(TheFooter, { shallow: true });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
