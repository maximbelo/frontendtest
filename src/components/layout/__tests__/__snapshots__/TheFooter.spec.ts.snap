// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TheFooter.vue tests > matches the snapshot 1`] = `
"<footer data-v-db5876b4=\\"\\">
  <div data-v-db5876b4=\\"\\" class=\\"container\\">
    <p data-v-db5876b4=\\"\\"> Â© 2023 maxim. </p>
  </div>
</footer>"
`;
